OneCommunityGlobal/HighestGoodNetworkApp
# Rishitha fix projects active filter

# Description
 (PRIORITY TBD) Jinxiong: The Projects page “Active” filter does not work correctly (WIP Rishitha)
Other Links>Projects>Active filter
Select Active or Inactive.
None of the records are displayed. It should displayed the filtered result.


## Related PRS (if any):
This frontend PR is related to the development backend PR.

## Main changes explained:
- Updated src/components/Projects/Projects.jsx for the filter to work

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to Other Links>Projects>Active filter
6. confirm if the active filter is working correctly

## Screenshots or videos of changes:

https://github.com/user-attachments/assets/8eb33d69-5dec-4dca-8f52-a39c08c9ec59
